CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.0.1] - 2024-05-14
+
+Props: @GaryJones, @jrnfl, @terriann, @rebeccahum
+
+Increases requirements for PHPCS from 3.7.2 to 3.9.2 for improved PHP 8.2 support. Please ensure you run `composer update automattic/vipwpcs --with-dependencies` to benefit from this.
```

Comment:
```suggestion
Increases requirements for PHPCS from 3.7.2 to 3.9.2 for improved PHP 8.2 and PHP 8.3 support. Please ensure you run `composer update automattic/vipwpcs --with-dependencies` to benefit from this.
```

PHPCS 3.9.0 added PHP 8.3 syntax support. One more PHP 8.2 one is still coming up in PHPCS 3.10.0, expected later this month.